### PR TITLE
[Tabs] Add Swift example for MDCTabBarView

### LINKED ISSUE #9068: [Tabs] Internal issue: b/145683119
This was filed as an internal issue. If you are a Googler, please visit [b/145683119](http://b/145683119) for more details.

---
#### Internal data
- Associated internal bug: [b/145683119](http://b/145683119)

### PR DESCRIPTION
**Pull request by bryanoltman**
Adds a rudimentary Swift example for `MDCTabBarView`

Fixes #9068

### CODE REVIEW

File: components/Tabs/examples/TabBarViewExample.swift [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+import MaterialComponentsBeta.MaterialTabs_TabBarView
+import MaterialComponents.MaterialTabs
```
Comment (romoore) [inline components/Tabs/examples/TabBarViewExample.swift]: Is this import needed? TabBarView shouldn't depend on Tabs.

File: components/Tabs/examples/TabBarViewExample.swift [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+class TabBarViewExample: UIViewController {
+
+  static let title = "TabBarView (Swift)"
+
+  @objc var containerScheme: MDCContainerScheming = MDCContainerScheme()
```
Comment (romoore) [inline components/Tabs/examples/TabBarViewExample.swift:24]: Please import MaterialContainerScheme

File: components/Tabs/examples/TabBarViewExample.swift [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    print("Item \(item.title!) was selected.")
+  }
+}
+
+extension TabBarViewExample {
+  @objc class func catalogMetadata() -> [String: Any] {
+    return [
+      "breadcrumbs" : ["Tab Bar", TabBarViewExample.title],
+      "primaryDemo" : false,
+      "presentable" : true,
```
Comment (romoore) [inline components/Tabs/examples/TabBarViewExample.swift]: This is a Beta component, so let's keep it in the Dragons for now.

**Review by romoore — APPROVED** [review]

**Review by romoore — CHANGES_REQUESTED** [review]
TabBar isn't visible on iPhone 4s/iOS 9.3 simulator. Shows for other TabBarView example.

|ObjC|Swift|
|---|---|
|Simulator Screen Shot - iPhone 4s - 2019-12-11 at 11 36 35|Simulator Screen Shot - iPhone 4s - 2019-12-11 at 11 34 44|

File: components/Tabs/examples/TabBarViewExample.swift (hunk 1 above)
Comment (bryanoltman) [reply components/Tabs/examples/TabBarViewExample.swift]: Nope! Will fix.

File: components/Tabs/examples/TabBarViewExample.swift (hunk 2 above)
Comment (bryanoltman) [reply components/Tabs/examples/TabBarViewExample.swift:24]: Done.

**Review by romoore — APPROVED** [review]

**Pull request merged**